Resources/doc/cookie_sync.rst

```diff
@@ -0,0 +1,177 @@
+Synchronizing Cart And Customer
+===============================
+
+Synchronizing from Magento to ONGR is done via cookies. After every action with cart (adding or removing products)
+a cookie is set with all the products in cart. Also after login, logout and customer information update another cookie
+is updated.
+
+Synchronization from ONGR to Magento is done via requests. More detailed information about will come up later.
+
+Setting up synchronization will require installing `MagentoSyncModule <https://github.com/ongr-io/MagentoSyncModule>`_
+and a bit of configuration.
+
+Step 1. Installing Magento Module
+---------------------------------
+
+Follow standard procedure for installing `Magento Sync Module <https://github.com/ongr-io/MagentoSyncModule>`_. Then,
+if needed, set domain of ONGR project in ``system->configuration->ONGR SYNC->Sync Options``. This setting is used for
+cookie domain and if left empty current domain will be used.
+
+Step 2. Configuring MagentoConnectorBundle
+------------------------------------------
+
+This bundle has 2 services to handle the synchronization with magento and they require some configuration to work
+properly.
+
+In your config file under ``ongr_magento`` you will need to set ``url`` of Magento store, ``es_manager`` and
+``product_repository``.
+
+Example configuration:
+
+.. code-block:: yaml
+
+    ongr_magento:
+        store_id: 0
+        shop_id: 1
+        url: http://magento.ongr.dev
+        es_manager: magento
+        product_repository: ONGRMagentoConnectorBundle:ProductDocument
+
+..
+
+Using the Customer service.
+---------------------------
+
+Customer service id is ``ongr_magento.sync.cart``. This service allows getting currently logged in customer information
+and helps with login and logout functionality.
+
+Login
+~~~~~
+
+Method ``getLoginUrl`` will give url to login page in Magento with back url parameter of current location so that user
```

Comment:
Method ``getLoginUrl`` will return url to login page in Magento with return url of current location attached as a parameter so that user ...